.github/workflows/test-pull-request.yml

```diff
@@ -64,6 +64,9 @@ jobs:
         echo "Success: Static site built in $SITE_DIR."
         exit 0
 
+    - name: Reduce size of debug artifact
+      run: rm -rf _site/media
+
```

Comment:
Artifact before this is about 8.16 MB (compressed) and takes about 30 seconds to upload. Now it is about 2.97 MB (compressed) and takes 7 seconds. If a test fail requires media to debug it can be commented out and reverted later. For now, this speed improvement is very noticable.